src/compaction_execute_bench/README.md

```diff
@@ -28,6 +28,9 @@ Set the following environment variables to configure the benchmark:
 
 `VAL_BYTES` : Size of values in bytes (default: 224).
 
+`COMPACTION` : Set to a comma-separated list of SR ids to compact SRs from an existing database instead of
```

Comment:
I think this should mention that the last ID is the destination ID. At least, that's how I read the code in `parse_compaction`